app/src/components/presentational/Atividades/AtividadesPresentational.js

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import styled from "styled-components/native";
+import { Colors, Spacing, Typography } from "../../../config";
+
+import {
+  AlignBallNotification,
+  BallNotification,
+  ContainerNotification,
+  ContainerInfoNotification,
+  ContainerDateNotification,
+  DateNotification,
+  DescNotification,
+  HeaderAtividades,
+  ListNotification,
+  TitleHeaderAtividades,
+  TitleNotification,
+  ScrollList,
+  Wrapper } from "./styles.js";
+
+import ModalAtividades from "./Modal";
+
+const ExemploNotificacoes = [
+  {
+    title: "01/12/2020",
+    data:  ["Uma anotação sua foi comentada"],
+    desc: ["Sua anotação “Arranjos e Contagens” do caderno “INF3 - Matemática” recebeu um comentário."]
+  },
+  {
+    title: "30/11/2020",
+    data:  ["Uma anotação sua foi comentada"],
+    desc: ["Sua anotação “Arranjos e Contagens” do caderno “INF3 - Matemática” recebeu um comentário."]
+  },
+];
+
+//Testar  onPress={()=> Alert.alert('Teste')}
+
+const TelaAtividades = () => (
+  <>
+    <Wrapper>
+    <HeaderAtividades>
+      <TitleHeaderAtividades>Atividades Recentes</TitleHeaderAtividades>
+        <ModalAtividades />
+      </HeaderAtividades>
+      <ScrollList>
+      <ListNotification
+            sections={ExemploNotificacoes}
+            keyExtractor={(item, index) => item + index}
```

Comment:
item é um objeto, por mais que não faça muita diferença, ficaria melhor:

```suggestion
            keyExtractor={(item, index) => index.toString()}
```
ou
```suggestion
            keyExtractor={(item, index) => item.id}
```

Reply:
Resolvido